### DISCUSSION TITLE
JSONrpc for quiteGroup

### DISCUSSION THREAD

**Discussion by CairbreMadog** [AsamK/signal-cli#1206] [Q&A]
I am trying to generate some JSON-rpc to quiteGroup:

This is what I am currently using: But it does not work: 

{"jsonrpc":"2.0","method":"quitGroup","params":{"name":"GroupName", "groupId":"group.ABCDEFG="},"id":4561}
or 
{"jsonrpc":"2.0","method":"quitGroup","params":{"groupId";"group.ABCDEFG="},"id":"56546}

**Comment by AsamK** [comment]
The second one is correct. Are you sure you're using the correct groupId? There should be no `group.` prefix.
